__tests__/components/menu/CanvasMenu.test.js

```diff
@@ -25,4 +25,8 @@ describe('<CanvasMenu />', () => {
       expect(node.prop('rel')).toEqual('noopener noreferrer')
     })
   })
+
+  it('has an aria-label for accessability', () => {
+    expect(wrapper.find('button[aria-label="Close Help Menu"]')).toBeTruthy()
```

Comment:
`toBeDefined()` seems like it'd be more consistent with the rest of the tests, and makes a bit more sense to me semantically, but `toBeTruthy()` also seems like it'd do the same job just fine in this case, so this isn't something i'd ask for changes over.

feel free to merge if you think the change isn't worth it, or if you do like the idea and wind up implementing it.
```suggestion
    expect(wrapper.find('button[aria-label="Close Help Menu"]')).toBeDefined()
```